monitoring/dashboard.json

```diff
@@ -16,35 +24,1214 @@
             "value": "zenko"
         },
         {
-            "name": "job",
-            "type": "constant",
-            "label": "job",
-            "description": "Name of the Cloudserver job, used to filter only the Cloudserver instances.",
-            "value": "artesca-data-connector-s3api-metrics"
-        }
-    ],
-    "editable": true,
-    "gnetId": null,
-    "graphTooltip": 0,
-    "id": 33,
-    "links": [],
-    "panels": [
+            "name": "job",
+            "type": "constant",
+            "label": "job",
+            "description": "Name of the Cloudserver job, used to filter only the Cloudserver instances.",
+            "value": "artesca-data-connector-s3api-metrics"
+        },
+        {
+            "name": "pod",
+            "type": "constant",
+            "label": "pod",
+            "description": "Prefix of the Cloudserver pod names, used to filter only the Cloudserver instances.",
+            "value": "artesca-data-connector-cloudserver"
+        }
+    ],
+    "editable": true,
+    "gnetId": null,
+    "graphTooltip": 0,
+    "id": null,
+    "links": [],
+    "panels": [
+        {
+            "datasource": "${DS_PROMETHEUS}",
+            "description": "",
+            "fieldConfig": {
+                "defaults": {
+                    "color": {
+                        "mode": "thresholds"
+                    },
+                    "mappings": [],
+                    "thresholds": {
+                        "mode": "absolute",
+                        "steps": [
+                            {
+                                "color": "red",
+                                "value": null
+                            },
+                            {
+                                "color": "green",
+                                "value": 1
+                            }
+                        ]
+                    }
+                },
+                "overrides": []
+            },
+            "gridPos": {
+                "h": 4,
+                "w": 3,
+                "x": 0,
+                "y": 0
+            },
+            "id": 20,
+            "options": {
+                "colorMode": "value",
+                "graphMode": "area",
+                "justifyMode": "auto",
+                "orientation": "auto",
+                "reduceOptions": {
+                    "calcs": [
+                        "lastNotNull"
+                    ],
+                    "fields": "",
+                    "values": false
+                },
+                "text": {},
+                "textMode": "auto"
+            },
+            "pluginVersion": "8.0.6",
+            "targets": [
+                {
+                    "exemplar": true,
+                    "expr": "sum(up{namespace=\"${namespace}\", job=\"${job}\"})",
+                    "instant": false,
```

Comment:
same here I believe the `instant=true` should be set, we only want the last non null value.

Reply:
not really : since we have the "background" graph : showing the history of this gauge